setup.py

```diff
@@ -1,14 +1,11 @@
-import os
 from setuptools import setup, find_packages
+import subprocess
 
 
 def get_version():
-    basedir = os.path.dirname(__file__)
-    with open(os.path.join(basedir, 'rq_dashboard/version.py')) as f:
-        locals = {}
-        exec(f.read(), locals)
-        return locals['VERSION']
-    raise RuntimeError('No version info found.')
+    """Return the latest git tag from repository in cwd."""
```

Comment:
So: how about we use setuptools_git_version instead then?

It looks like it falls back to using the pkg_resource version if it has not been checked out in a git repo: https://github.com/pyfidelity/setuptools-git-version/blob/master/setuptools_git_version.py

I'm going to give it a quick go...